packages/react/src/progress-indicator/ProgressIndicator.tsx

```diff
@@ -34,49 +51,82 @@ export const ProgressIndicator = ({
 				items.length
 		  } steps completed`;
 
-	const hasExplicitActiveDeclared =
-		items.filter(({ isActive }) => isActive).length > 0;
+	/* **LEGACY HANDLER**
+	 * This is now legacy, please keep this for backwards compatibility.
+	 */
+	const hasExplicitActiveDeclared = items.some(({ isActive }) => isActive);
+
+	if (process.env.NODE_ENV !== 'production' && hasExplicitActiveDeclared) {
+		console.warn(
+			'ProgressIndicator: The `isActive` key is deprecated. Please use `activePath` in the parent component, `<ProgressIndicator />`.'
```

Comment:
This feels a bit clearer
```suggestion
			'ProgressIndicator: The `isActive` key is deprecated. Please set `activePath` in the `<ProgressIndicator />`.'
```